### ISSUE TITLE
[Raspberry Pi] wget sodium for amr7

### ISSUE THREAD

**Issue by munakaya** [tezos/tezos#168]
Hi, I am trying to install tezos alpha net on raspberry Pi 3. I installed ubuntu MATE and built tezos. But I faced sodium.dev build fail. 

From x86 build, I solved this problem by installing sodium by wget. 
( I am running tezos node on x86/MAC )

Anyone can let me know how to wget "arm7" sodium for raspberry Pi ?
I want to know the location of sodium which built for arm7.

**Comment by maxtez-raspbaker** [comment]
Hello, I am not part of the Tezos development team but perhaps I can help since I compiled Tezos (alphanet branch) on RPI3 (more info here: forums.tezos.community). With Fedora distro I simply used:
dnf install libsodium-devel
Ubuntu repository should have a similar package

**Comment by munakaya** [comment]
Hi, I am writing easy way to install tezos node on RPI3 for not trained people. Tezos build on Fedora is so complicated.

**Comment by maxtez-raspbaker** [comment]
Yes please write some notes and share your intel. I am also planning to revise my notes at some point. In the meantime there is an image of fedoraOS+precompiled Tezos for the alphanet that can be downloaded. Just write the image on the SD card and the RPI3 is ready to run the node (info here: https://forums.tezos.community/)
